### ISSUE TITLE
[Feature Request] Additional cooling fan options 

### ISSUE THREAD

**Issue by cheronobyl** [teachingtechYT/teachingtechYT.github.io#63]
It would be helpful to have more options for cooling fan settings in all of the gcode generators. Being able to set a percentage @ a layer would assist people with less standard build surfaces.

**Comment by teachingtechYT** [comment]
This is useful although I'd still prefer some presets to preserve sanity in implementation.
I assume this is mainly for PLA and filaments where the fan will be on the entire print?

**Comment by cheronobyl** [comment]
Yes, my specific use case is to have the fan come on at later levels when printing with PLA on my PEX-equipped WhamBam. Their literature suggests 3-4 layers of no fan speed and I'm having issues only having it off during the first layer.

**Comment by teachingtechYT** [comment]
Ok I'll try and get this one in over the next few days.
